pkgs/development/python-modules/aenum/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "aenum";
+  version = "1.4.7";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
```

Comment:
@grahamc `fetchPypi` is a new helper, taking `pname`, `version`, and optionally `format` along with some extra arguments in case `format = "wheel"`.

New expressions are preferably put in separate files in `python-modules`. `python-packages.nix` is getting too big. This is correct now @vrthra